### DISCUSSION TITLE
Private, dark networks for APIs

### DISCUSSION THREAD

**Discussion by surennaidu** [apache/apisix#9333] [General]
I'm part of the leadership team at [NetFoundry](www.netfoundry.io) and we came across your solution. We help organizations move their API traffic away from the internet to private zero trust overlay networks. Would like to know if your customers would care if APIs are protected from the internet; if yes, we can talk.[ Here is a brief about our API security offering.](https://netfoundry.io/zero-trust-api-clouds/)

**Comment by philipleonardgriffiths** [comment]
Note, NetFoundry is built on top of open source OpenZiti. We took our C SDK and created a module for Nginx. The exact same thing could be done for APISIX API Gateway https://blog.openziti.io/nginx-zerotrust-api-security.
